packages/cluster-ui/src/statementsPage/statementsPage.module.scss

```diff
@@ -53,6 +53,10 @@ h2.base-heading {
   width: 100%;
 }
 
+.table-area {
```

Comment:
nit: this is what fixes the resize bug, right? I would've put this in a separate commit as well but don't worry about it now.

Reply:
that's correct, it was for the resizing. I'll keep in mind for future commits :)